directory.json

```diff
@@ -42,6 +42,7 @@
 	"HLUG": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/China/Hangzhou(%E6%9D%AD%E5%B7%9E)LinuxUserGroup-api.json",
 	"HSG": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Singapore/HackerspaceSG",
 	"IDR": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Indonesia/IDRuby-api.json",
+	"IT": "https://raw.githubusercontent.com/nolik2000/fossasia-communities/master/Sri%20Lanka/MillenniumIT-api.json",
```

Comment:
@nolik2000 Follow the pattern as mentioned in the [README](https://github.com/fossasia/directory.api.fossasia.net/blob/master/README.md).
I think it should go like `"IT": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/SriLanka/MillenniumIT-api.json"`